tests/unit/utils/test_gmm.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import numpy as np
+from nequip.utils import gmm
+from sklearn import mixture
+
+fit_data = np.random.rand(10, 3)
+gmm_sklearn = mixture.GaussianMixture(
+    n_components=3, covariance_type="full", random_state=123
+)
+
+gmm_torch = gmm.GaussianMixture(n_components=3, seed=123, n_features=3)
```

Comment:
Use pytest fixtures to construct common data or objects used in multiple tests:  https://docs.pytest.org/en/7.1.x/explanation/fixtures.html#about-fixtures